explore/src/main/scala/explore/attachments/ObsAttachmentsTableBody.scala

```diff
@@ -60,17 +60,22 @@ import org.typelevel.log4cats.Logger
 
 import scala.collection.immutable.SortedSet
 
-case class ObsAttachmentsTable(
+object ObsAttachmentsTableTileState extends NewType[Action]:
+  def apply(): ObsAttachmentsTableTileState = ObsAttachmentsTableTileState(Action.None)
+type ObsAttachmentsTableTileState = ObsAttachmentsTableTileState.Type
+
+case class ObsAttachmentsTableBody(
```

Comment:
Maybe we want to keep the name of the file `ObsAttachmentsTable` (without `Body`).

Also, have you considered the following structure?
``` scala
object ObsAttachmentsTable:
  object State extends NewType[Action]
  
  case class Body(...)
  object Body {
    ...
  }
  
  case class Title(...)
  object Title {
    ...
  }
```

Reply:
Seems good, maybe calling it `ObsAttachmentsTile`?